compiler/src/dmd/frontend.h

```diff
@@ -5948,6 +5950,10 @@ class VarDeclaration : public Declaration
     bool isArgDtorVar(bool v);
     bool isCmacro() const;
     bool isCmacro(bool v);
+    bool inClosure() const;
+    bool inClosure(bool v);
+    bool inAlignSection() const;
+    bool inAlignSection(bool v);
```

Comment:
These will cause linker errors for gdc and ldc.

Reply:
Nobody uses this header.

Reply:
That's why frontend.h, being a generated file, should not get checked in.